Objects/stringlib/eq.h

```diff
@@ -3,7 +3,7 @@
 /* Return 1 if two unicode objects are equal, 0 if not.
  * unicode_eq() is called when the hash of two unicode objects is equal.
  */
-Py_LOCAL_INLINE(int)
+__attribute__((always_inline)) Py_LOCAL_INLINE(int)
```

Comment:
Just curious, why do we need this?

Reply:
I just rechecked this and I'm getting this error without it:
```
/usr/bin/ld: aot_optimized.o: in function `lookdict_unicode_nodummy':
/home/undingen/dev/pyston/build/bc_build/../../Objects/dictobject.c:892: undefined reference to `unicode_eq'
```
Seems like on ARM the compiler makes a bit different inlining decision.

Reply:
And the problem is that this function is defined in a header which get included from `dictobject.c` and `unicodeobject.c` So just removing the static will make it complain about having it defined twice.

Reply:
Ah I see, yeah I think removing `static` has been our more typical approach so I wasn't sure why that wasn't used here, but now I get it